CYOAfeb21

```diff
@@ -0,0 +1,225 @@
+###Standard Question Setup###
+def multich(ch1="error1", ch2="error2", ch3="error3", ch4="error4"):
+	choice= input(f"""
+	Select Option:
+		A. {ch1}
+		B. {ch2}
+		C. {ch3}
+		D. {ch4}""")
+	return choice
+	
+###Questions for naming and selecting play class.###
+def givename():
+	name= input(print("Type your name."))
+	return name
+def giveclass():
+	playclass= multich(ch1='Warrior', ch2='Wizard', ch3='Cleric', ch4='Rogue')
+	return playclass
+	#add details about stats here
+#name, position, items, attack, defense, health, magic, experience, level, playclass
+
+###Define character class###
+class Character:
+	def __init__(self, name, position, items):
+		self.name= name
+		self.position= position
+		self.items= items
+#Setters and Getters for all Characters
+#Name		
+	def name(self):
+		return self.name
+	def name(self, value):
+			self.value= value
+#Position			
+	def position(self):
+		return self.position
+	def position(self, value):
+		self.value= value
+#Items			
+	def items(self):
+		return self.items
+	def items(self, value):
+		self.value= value
+###//Define NPC Subclass//###
+###Not needed since they only need character parameters###
+
+###Start NPC Bob###
+bob= Character('Bob', 'Town', 'Main Key')
+#Function for printing bob.
+def bobsheet():
+	print(f"""
+	Name: {bob.name}
+	Position: {bob.position}
+	Items: {bob.items}""")
+#Printing Bob
+#bobsheet()	
+	
+###Define Battlers###
+class Battler(Character):
+	def __init__(self, name, position, items, attack, defense, health, magic, experience, level, playclass):
+		self.name= name
```

Comment:
You don't want to duplicate this code. Do the thing in the other branch that initializes a super class instead.

`super().__init__(name = name, position=position, items=items)`

Reply:
I thought I had done that subclass thing. but it didnt run without the duplicated code. I need to look more closely apparently.